### ISSUE TITLE
In the latest mbed-os, some message for TLS certificate is not been printing out anymore

### ISSUE THREAD

**Issue by jamesbeyond** [ARMmbed/mbed-os-example-tls-socket#51]
in the latest mbed-os 

the message of
```
[DBG ][TLSW]: mbedtls_ssl_conf_ca_chain()
[DBG ][TLSW]: mbedtls_ssl_setup()
[DBG ][TLSW]: Server certificate:
    cert. version     : 3
    serial number     : 0C:41:8C:DF:D9:3E:0D:A0:FC:7F:46:90:C1:64:A0:F2
    issuer name       : C=US, ST=CA, L=San Francisco, O=CloudFlare, Inc., CN=CloudFlare Inc ECC CA-2
    subject name      : C=US, ST=CA, L=San Francisco, O=Cloudflare, Inc., CN=sni.cloudflaressl.com
    issued  on        : 2020-01-30 00:00:00
    expires on        : 2020-10-09 12:00:00
    signed using      : ECDSA with SHA256
    EC key size       : 256 bits
    basic constraints : CA=false
    subject alt name  :
        dNSName : sni.cloudflaressl.com
        dNSName : *.ifconfig.io
        dNSName : ifconfig.io
    key usage         : Digital Signature
    ext key usage     : TLS Web Server Authentication, TLS Web Client Authentication
```

all the DBG messages have been swallowed. 
if  that is the desired change, both
https://github.com/ARMmbed/mbed-os-example-tls-socket/blob/master/README.md
https://github.com/ARMmbed/mbed-os-example-tls-socket/blob/master/tests/tls-socket.log
need to be updated

**Comment by jamesbeyond** [comment]
Hi @AnttiKauppila @kivaisan, any ideas on this issue?

**Comment by ciarmcom** [comment]
*Internal Jira reference: https://jira.arm.com/browse/IOTCELL-2450*

**Comment by kivaisan** [comment]
@jamesbeyond The reason is the default trace level has been changed to INFO: https://github.com/ARMmbed/mbed-os/commit/7b0ac0669fdf093f6a4df1a08c660d59592a0601#diff-d51003071c3a7eb85269b89d98e191dd

So either change the log comparision, or change trace level back to DEBUG for this application. The easiest way to do it is to change it in `mbed_app.json` with `"mbed-trace.max-level": "TRACE_LEVEL_DEBUG"`

**Comment by kivaisan** [comment]
Fixed with 60d54b62fc618a329c0eafc65301650c66d70ed6
